core/arch/arm/include/mm/core_mmu_arch.h

```diff
@@ -210,6 +210,17 @@ static inline unsigned int core_mmu_get_va_width(void)
 	}
 	return 32;
 }
+
+static inline bool core_mmu_table_level_in_range(unsigned int level)
```

Comment:
I'm sorry for picking on this again, but I'd prefer to use the name `core_mmu_level_in_range()`.

Reply:
That's OK~
Changes and tags are included in new commits.